charts/jaeger/values.yaml

```diff
@@ -2,30 +2,15 @@
 # This is a YAML-formatted file.
 # Jaeger values are grouped by component. Cassandra values override subchart values
 
-global:
```

Comment:
please minimize the amount of changes in the PR. It looks like you're not really changing this section, but it's being moved to the end so makes it difficult to compare / review.